client/src/reducers/websiteReducer.js

```diff
@@ -1,23 +1,50 @@
 /**
- * This file exports website reducer that will handle all dispatched website-related actions.
+ * This file exports our team reducer that will handle all dispatched team-related actions.
  */
- import {CREATE_WEBSITE, DELETE_WEBSITE} from "../actions/types";
+import {
+    CREATE_WEBSITE,
+    ADD_WEBPAGE,
+    DELETE_WEBPAGE,
+    FETCH_WEBSITE_INFO,
+  } from '../actions/types';
+  
+  const INITIAL_WEBSITE_STATE = {
+    url: '',
+    title: '',
+    pages: [],
+    availablePages: ['PUBLICATIONS', 'TEAM'],
```

Comment:
You can pull the availablePages up into /config so it'll be easier to maintain when we need to add more pages

Reply:
Great idea! I didn't know config existed lol